pkg/controllers/podgroup/pg_controller.go

```diff
@@ -163,9 +163,13 @@ func (pg *pgcontroller) processNextReq() bool {
 		return true
 	}
 
-	if pod.Annotations != nil && pod.Annotations[scheduling.KubeGroupNameAnnotationKey] != "" {
-		klog.V(5).Infof("pod %v/%v has created podgroup", pod.Namespace, pod.Name)
-		return true
+	if pod.Annotations != nil {
+		if pgName := pod.Annotations[scheduling.KubeGroupNameAnnotationKey]; pgName != "" {
```

Comment:
why split it

Reply:
It is recommended to describe the current problem encountered, thanks. @elinx